README.md

```diff
@@ -134,6 +134,13 @@ functions:
 * Day cold periods
 * Desire to avoid cold lambdas after a deployment
 
+#### Runtime Configuration
+Concurrency can be modified post-deployment at runtime by using Lambda environment variables.  
+Two configuration options exist:
+* Globally set the concurrency of all lambdas on the stack (overriding the deployment configuration):  
+  Set the environment variable `WARMUP_CONCURRENCY`
+* Individually set the concurrency per lambda  
+  Set the environment variable WARMUP_CONCURRENCY_YOUR_FUNCTION_NAME. Must be all uppercase and hyphens (-) are replaced with underscores (_) 
```

Comment:
Can you add a note saying something along the lines of `If present, it overrides the global concurrency setting.`